docs/ossm/ossm2-vs-ossm3.md

```diff
@@ -0,0 +1,103 @@
+# Compared to OpenShift Service Mesh 2
+
+OpenShift Service Mesh 3 includes many important differences to be aware of coming from OpenShift Service Mesh 2.
+
+OpenShift Service Mesh 3 is a major update with a feature set closer to the [Istio](https://istio.io/) project. It is based directly on Istio, rather than the midstream Maistra project that OpenShift Service Mesh 2 was based on. It is managed using a different, simplified operator and provides greater support for the latest stable features of Istio. This alignment with the Istio project along with lessons learned in the first two major releases of OpenShift Service Mesh have resulted in the following changes.
+
+## From Maistra to Istio
+
+While OpenShift Service Mesh 1 and 2 releases were based on Istio, they included additional functionality that was maintained as part of the midstream Maistra project, which itself was based on the upstream Istio project. Maistra maintained several features that were not part of the upstream Istio project. While this provided extra features to OpenShift Service Mesh users, the effort to maintain the Maistra meant that OpenShift Service Mesh was usually several releases behind Istio with support omitted for major features like multi-cluster. Meanwhile, Istio has matured to cover most of the use cases addressed by Maistra. Basing OpenShift Service Mesh directly on Istio ensures that it will support users on the latest stable Istio features while Red Hat is able to contributing directly to the Istio community on behalf of its customers. 
```

Comment:
```suggestion
While OpenShift Service Mesh 1 and 2 releases were based on Istio, they included additional functionality that was maintained as part of the midstream Maistra project, which itself was based on the upstream Istio project. Maistra maintained several features that were not part of the upstream Istio project. While this provided extra features to OpenShift Service Mesh users, the effort to maintain Maistra meant that OpenShift Service Mesh was usually several releases behind Istio with support omitted for major features like multi-cluster. Meanwhile, Istio has matured to cover most of the use cases addressed by Maistra. Basing OpenShift Service Mesh directly on Istio ensures that it will support users on the latest stable Istio features while Red Hat is able to contribute directly to the Istio community on behalf of its customers. 
```